openlibrary/templates/search/sort_options.html

```diff
@@ -38,31 +38,20 @@
        ]
        if ctx.user and (ctx.user.is_admin() or ctx.user.is_librarian()):
          sort_options.append({ 'sort': 'title', 'name': _("Work Title (beta: Librarian only)"), 'ga_key': 'Title' })
+    <select class="generic-dropper-wrapper" onchange="window.location.href = this.value">
```

Comment:
Rather than using javascript -- what if the patron has js disabled? -- I wonder if it may make more sense for us to use a `details` component and have each item in the component be a clickable link.

This was mentioned as an idea in the original issue description: https://github.com/internetarchive/openlibrary/issues/8829#issue-2145193292

Here's a codepen where we implement something similar using the `details` html element type:

https://codepen.io/sivanc/pen/VwOpXwq?editors=1100